domestic/melodic/Dockerfile

```diff
@@ -21,6 +21,6 @@ WORKDIR /melodic
 
 RUN catkin config --init && \
     catkin config --extend /opt/ros/melodic && \
-    catkin config --install --install-space /opt/ros/mas_stable && \
-    catkin build -c && \
```

Comment:
Why were these changes needed? Also, as far as I know, each individual `RUN` statement adds a layer to the image and therefore increases its size